src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1708,71 +1708,128 @@ Below is an example for adding a certificate to a Java cacerts file:
+`Append to current Workflow`
+When the user choose this option, it's supposed that the user wants to append the workflow from catalog to workflow already opened inside studio. Please open the workflow to which you want to append before.
+
+=== Workflow Catalog Portal
+The Workflow Catalog Portal provides interfaces for easy administering the catalog. The most important of provided features will be described next.
+
+==== Creating the new buckets
+You need to press `plus` button from above of buckets. In opened window you need to specify Bucket name and associate with user group.
+For security reason there are groups associated to buckets. The user from same group can view the content of bucket.
+There is a special default group - `no group` in the dropdown list of window. The bucket associated with this default group can be viewed by any user.
+
+==== Removing workflows inside bucket
+For removing workflows, first the user should select the bucket containing the workflows. Then the user can select multiple workflows with pressed CTRL button. To delete the selected workflows push the `trash` button.
 
-Evaluation priority is respected through operator precedence (`AND` over `OR`), so the following statement:
+===== Importing/Exporting Workflows into the Catalog
+In the Workflow Catalog, it is possible to Export Workflows as an archive, then Import them.
+You can export Workflows into an archive, which can be transferred by any file transfer method (email, usb key, etc.) and then imported into the other Scheduler.
 
-    Search for all Workflows that can deploy an OpenStack instance
+To export the selected Workflows as a ZIP archive from Workflow catalog portal: select a bucket, then the targeted Workflows (you can select multiple workflows with pressed CTRL button). Press "Download Workflows" button from portal view. You'll get a ZIP file which can be transferred to the machine on which the other Scheduler runs.
+To import the zip archive of workflows to the Catalog from portal, select the bucket to which the Workflows will be added. Then press `Upload Workflows` button in portal and follow the instructions `Upload a ZIP archive of workflows` opened window.
 
-can be expressed as:
+==== List revisions of workflow
+In portal you always see the last revision of catalog. In order to see all revisions of selected workflow, please click on `clock` button inside description area.
+The revisions are shown in chronological order by commit time.
 
-    name="Create OpenStack Instance" OR project_name="Deployment" AND generic_information("Infrastructure", "OpenStack")
+==== Restore the revision
+Usually the users works the last version of workflow. But there is the case when the user wants to make the specific revision as the last version of workflow.
+For this case there is a special option inside portal: `Restore the revision`. The user should select the workflow, click `clock` button to see the list of all revisions.
```

Comment:
.. there is a dedicated button: ... Once the workflow is selected, click on the 'clock' button to expand all revisions.